src/openforms/payments/views.py

```diff
@@ -307,19 +318,47 @@ def finalize_response(self, request, response, *args, **kwargs):
     ],
     request=None,
     responses={
-        200: None,
-        (400, ERR_CONTENT_TYPE): OpenApiResponse(
+        (status.HTTP_200_OK, "text/plain"): OpenApiResponse(
+            response=str,
+            examples=[
+                OpenApiExample(
+                    name="Webhook event",
+                    value="",
+                    media_type="text/plain",
+                    status_codes=[status.HTTP_200_OK],
+                    description=_(
+                        "The response returned when a webhook event is processed."
+                    ),
+                ),
+                OpenApiExample(
+                    name="Verification header",
+                    value="<verification-header-value>",
+                    media_type="text/plain",
+                    status_codes=[status.HTTP_200_OK],
+                    description=_(
+                        "The response returned when a verification header is set"
+                        " and no request body is found."
+                    ),
+                ),
+            ],
+        ),
+        (status.HTTP_400_BAD_REQUEST, ERR_CONTENT_TYPE): OpenApiResponse(
             response=ExceptionSerializer,
             description=_("Bad request. Invalid parameters were passed."),
         ),
-        (404, ERR_CONTENT_TYPE): OpenApiResponse(
+        (status.HTTP_404_NOT_FOUND, ERR_CONTENT_TYPE): OpenApiResponse(
             response=ExceptionSerializer,
             description=_("Not found. The slug did not point to a live plugin."),
         ),
     },
 )
 class PaymentWebhookView(PaymentFlowBaseView):
-    parser_classes = (FormParser, MultiPartParser)
+    parser_classes = (
+        FormParser,
+        MultiPartParser,
+        CamelCaseJSONParser,
+        PlainTextParser,
+    )  # TODO: should we derive this from the plugin?
```

Comment:
Should we apply whats mentioned in the TODO?

Reply:
[I added developer documentation](https://github.com/open-formulieren/open-forms/pull/5498/commits/cdece166a64550a6f34bb3b873d332c7d985b548) on how I got this working locally.